src/main/java/com/blamejared/crafttweaker/impl/brackets/BracketHandlers.java

```diff
@@ -403,8 +422,48 @@ public static VillagerProfession getProfession(String tokens) {
             CraftTweakerAPI.logError("Could not get profession <profession:%s>", tokens);
             return null;
         }
-        
+    
         return ForgeRegistries.PROFESSIONS.getValue(resourceLocation);
     }
     
+    /**
+     * Gets a Biome based on the tokens.
+     * Throws an error if it can't get such a biome
+     *
+     * @param tokens The biome resource location
+     *
+     * @return The found biome
+     *
+     * @docParam tokens "minecraft:plain"
+     */
+    @ZenCodeType.Method
+    @BracketResolver("biome")
+    public static Biome getBiome(String tokens) {
+        
+        final int length = tokens.split(":").length;
+        if(length != 2) {
+            throw new IllegalArgumentException("Could not get biome with name: <biome:" + tokens + ">! Syntax is <biome:modid:biomeName>");
+        }
+        final ResourceLocation resourceLocation = new ResourceLocation(tokens);
+        if(!ForgeRegistries.BIOMES.containsKey(resourceLocation)) {
+            throw new IllegalArgumentException("Could not get biome <biome:" + tokens + ">");
+        }
+        return ForgeRegistries.BIOMES.getValue(resourceLocation);
+    }
+    
+    /**
+     * Gets damage source based on type.
+     * If the damage source is not pre-registered, it will create a new one with the given name
+     *
+     * @param tokens the damage sources' type
+     *
+     * @return The found pre-registered damage source or a new one
+     *
+     * @docParam tokens "magic"
+     */
+    @ZenCodeType.Method
+    @BracketResolver("damageSource")
```

Comment:
Ideally we want all BracketHandlers to be lowercase, so `damagesource` and `equipmentslottype`